integration_tests/jira_issues_tests.feature

```diff
@@ -72,6 +72,15 @@ Feature: Jira Issues Tests
     When we run a multi-day d3a simulation with jira.d3asim_962 [2019-01-01, 48, 60, 60]
     Then the device statistics are correct
 
+  @slow
+  # TODO: this is to be fixed as it is always failing
```

Comment:
Hmm, so no easy fix for that? Did you try to find out what goes wrong here?

Reply:
@hannesdiedrich it was a problem with the floating point precision, it was not equal to 0 but very close to it, fixed now

Reply:
great!